### DISCUSSION TITLE
5mods page

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1438] [Ideas]
Not exactly this repo related but it would be nice if you created a profile on [5mods](https://www.gta5-mods.com/) and created an upload (or somehow took over the old one) for SHVDN there since it is now [some half-assed one](https://www.gta5-mods.com/tools/scripthookv-net) running from crosire's suspended account. I'm sure the mods will quickly link the download button of it to the SHVDN repo, just like the original one is.

**Comment by kagikn** [comment]
5Mods doesn't allow to transfer ownerships of mods pages with the existing contents of stats intact. https://discord.com/channels/318621297057988608/318621297057988608/1110808965253709864

We have two options to ~~update downloads~~ push notifications on 5Mods, ask one of the moderators to do every time pushing notifications is needed, or ask them to delete the old one first and create a brand new mod page. I think I would try the former option first.

**Comment by Michael21107** [comment]
understood 
although the page just links to all releases, i think it would be nice if you created your own (requesting the deletion of the old one in the process or possible hard redirect from it) so you can have full control over let's say description

**Comment by justalemon** [comment]
If I'm not wrong, moderators can set mods to be redirects instead of file downloads. [OpenIV does this](https://www.gta5-mods.com/tools/openiv).

**Reply by Michael21107** [reply]
you are right indeed

**Comment by kagikn** [comment]
I forgot SHVDN's download button on 5Mods already redirects to this repo, even right before someone compromised crosire's 5Mods account. Still, we have to ask mods to send notifications for SHVDN updates on 5Mods or delete the old mod page though.

**Reply by Michael21107** [reply]
I dont know how can you make proper update notifications, probably only if it would be possible to change the link yourself and it would send a notif. But regarding the deletion of the old mod page, I'm sure they would do that without a problem and I'm all for it.

**Reply by Michael21107** [reply]
@kagikn any thoughts? I think it could be nice to implement before 3.7.0

**Reply by kagikn** [reply]
If what you said is about update notifications on 5Mods, I think manually asking a moderator should be enough to be honest. Other reasons could definitely justify recreating the mod page on 5Mods even if moderators can't transfer stats or comments.

**Reply by Michael21107** [reply]
my point was that you would have completely own and only SHVDN page there with a full account and therefore you would also be able to push updates how you want

**Reply by kagikn** [reply]
okay, fair, but i'm not doing that until the stable v3.7.0 is about to be released. Having a new page would allow me to control the description, so it would be worth it.

**Reply by Michael21107** [reply]
sounds good
